yesod-core/Yesod/Core/Handler.hs

```diff
@@ -780,6 +781,14 @@ deleteCookie a = addHeaderInternal . DeleteCookie (encodeUtf8 a) . encodeUtf8
 setLanguage :: MonadHandler m => Text -> m ()
 setLanguage = setSession langKey
 
+-- | Set attachment file name.
```

Comment:
Can you add a ChangeLog entry, cabal version bump, and a `@since` comment?

Reply:
I thinked that wait other pull request.
However, If I should write before it, I write.
So I write.